translations/ui-bulk-edit/en.json

```diff
@@ -314,9 +315,7 @@
   "layer.column.actions": "Actions",
   "layer.action.replace": "Replace with",
   "layer.action.clear": "Clear field",
-  "layer.action.apply.ITEMS": "Apply to items records",
-  "layer.action.apply.APPLY_TO_ITEMS": "Apply to all items records",
-  "layer.action.apply.APPLY_TO_HOLDINGS": "Apply to all holdings records",
+  "layer.action.applyItems": "Apply to items records",
```

Comment:
@UladzislauKutarkin, I don't understand how these translation-key changes are related to the code changes, given there is no overlap between the changing code and the changing keys. Can you explain?

Reply:
@zburke , sorry it's my bad. Already fixed it!